pkg/doc/verifiable/test-suite/config.json

```diff
@@ -1,8 +1,8 @@
 {
   "generator": "aries-framework-go-gen",
-  "presentationGenerator": "aries-framework-go-gen",
+  "presentationGenerator": "aries-framework-go-gen --presentation",
```

Comment:
`generatorOptions` is there precisely for passing parameters to the `generator` and `presentationGenerator`

Reply:
On the one hand, we would like to use the same generator for both Verifiable Credential and Verifiable Presentation testing. One the other hand, we need to distinguish the processing of VC from VP in the generator. That's why I put `--presentation` there for VP.